migrations/migrations/20210526193052-name-of-migration.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars */
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```

Comment:
you need to create a migration with the user schema, in order to creat the user table in the database